### PR TITLE
Add metric to keep track of systemd machine ID

### PR DESCRIPTION
**Pull request by lnovara**
The aim of this PR is to add a metric to keep track of machine ID and an alert that would fire whenever two or more nodes share the same machine ID.

Summary of changes:
  - enabled node_exporter's textfile collector
  - added `systemd_machine_id` metric
  - added `NodeMachineIDCollision` alert.

Closes: sighupio/fury-kubernetes-monitoring#21

### CODE REVIEW

**Comment by lzecca78** [comment]
awesome @lnovara ! 
thanks for this enhancement 👍

**Comment by lnovara** [comment]
Fixed pipeline.

**Comment by angelbarrera92** [comment]
Have we tested it?

File: katalog/node-exporter/deploy.yml [hunk 1]
```diff
@@ -13,6 +13,17 @@ spec:
       labels:
         app: node-exporter
     spec:
+      initContainers:
```
Comment (angelbarrera92) [inline katalog/node-exporter/deploy.yml]: awesome

**Comment by lnovara** [comment]
> Have we tested it?

I created a test cluster with our E2E environment, the new metric is correctly registered by node-exporter and the alert works as intended.

**Comment by angelbarrera92** [comment]
I will write the release docs for this change and then we can merge it ;)

**Pull request merged**